hr_attendance_geolocation/static/src/js/geolocation_user_context_legacy.esm.js

```diff
@@ -0,0 +1,31 @@
+/** @odoo-module **/
+/* Copyright 2023 Alexandre D. Díaz - Grupo Isonor
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+import {registry} from "@web/core/registry";
+import * as legacySession from "web.session";
+
+const LegacyGelocationSessionUserContextService = {
+    dependencies: ["user"],
+
+    start(env) {
+        Object.defineProperty(legacySession, "user_geolocation_context", {
+            set: (values) => {
+                const {latitude, longitude} = values;
+                if (typeof latitude !== 'undefined' && typeof longitude !== 'undefined') {
```

Comment:
Why not reliying on type coercion?

Reply:
Because 0 (zero) should be a valid value... in case of error the value is setted to 0.

Reply:
Sure :sweat_smile: :+1: